docs/src/get-started.md

```diff
@@ -0,0 +1,191 @@
+# Getting Started
+
+## Installation
+
+To use Turing, you need install Julia first and then install Turing.
+
+### Install Julia
+
+You will need Julia 0.6, which you can get from [the official Julia website](http://julialang.org/downloads/).
+
+It provides three options for users
+
+1. A command line version [Julia/downloads](http://julialang.org/downloads/) (**recommended**)
+2. A community maintained IDE [Juno](http://www.junolab.org/)
+3. [JuliaBox.com](https://www.juliabox.com/) - a Jupyter notebook in the browser
+
+For command line version, we recommend that you install a version downloaded from Julia's [official website](http://julialang.org/downloads/), as Turing may not work correctly with Julia provided by other sources (e.g. Turing does not work with Julia installed via apt-get due to missing header files).
+
+Juno also needs the command line version installed. This IDE is recommended for heavy users who require features like debugging, quick documentation check, etc.
+
+JuliaBox provides a pre-installed Jupyter notebook for Julia. You can take a shot at Turing without installing Julia on your machine in few seconds.
+
+### Install Turing.jl
+
+Turing is an officially registered Julia package, so the following should install a stable version of Turing:
+
+```julia
```

Comment:
These should all be updated to use the package manager REPL interface with `]`

Reply:
Generally this should just be `add Turing` to get whatever the latest release is, as opposed to `add Turing#master`. People should really on use `dev Turing` if they're interested in doing development work.